src/user/room.ts

```diff
@@ -428,12 +428,31 @@ export class Room extends Accessory implements Sayable {
     textOrContactOrFileOrUrl : string | Contact | FileBox | UrlLink,
     mention?                 : Contact | Contact[],
   ): Promise<void> {
+
+    const getRoomAlias: (c: Contact) => Promise<string> =
+    async (c: Contact) => {
+      try {
+        const alias: string | null = await c.alias()
```

Comment:
Why you are using `c.alias()` here?

I believe what you really want is `room.alias(c)`

Reply:
yes... I forget the new version of room alias.